starters/features/partytown/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "description": "Use Partytown into your Qwik app",
+  "devDependencies": {
+    "@builder.io/partytown": "^0.7.0"
+  },
+  "__qwik__": {
+    "priority": -1,
+    "viteConfig": {
+      "imports": [
+        {
+          "namedImports": [
+            "partytownVite"
+          ],
+          "importPath": "@builder.io/partytown/utils"
+        },
+        {
+          "namedImports": [
+            "join"
+          ],
+          "importPath": "path"
+        }
+      ],
+      "viteConfig": {
+        "ssr": "{ target: 'webworker', noExternal: true }"
```

Comment:
Not sure this line is needed

Reply:
I will try without this but i think is need

Reply:
I will try without this but i think is need

Reply:
Done!